cmd/helm/init_test.go

```diff
@@ -67,7 +67,7 @@ func TestInitCmd(t *testing.T) {
 	if !actions[1].Matches("create", "services") {
 		t.Errorf("unexpected action: %v, expected create service", actions[1])
 	}
-	expected := "Tiller (the helm server side component) has been installed into your Kubernetes Cluster."
+	expected := "Tiller (the helm server-side component) has been installed into your Kubernetes Cluster."
```

Comment:
- [ ] While we're still pursuing these corrections, why is "helm" written in lowercase here?  
  Did we miss that in the _init.go_ file?